exercises/03-sagas/README.md

```diff
@@ -127,15 +127,31 @@ class ShippingSagaData : ContainSagaData
 }
 ```
 
+NOTE: One could also use a property of type `int[]` for holding product identifiers as it's supported by the SqlPersister used in this exercise. 
```

Comment:
Maybe we should go ahead and change the type instead? Otherwise we should really justify our use of `ICollection<int>` over `int[]`.

Reply:
I'm 👍 for using `int[]`

Reply:
IMO you can go ahead and make that change instead.